addons/account/models/reconciliation_widget.py

```diff
@@ -543,6 +554,7 @@ def _domain_move_lines_for_reconciliation(self, st_line, aml_accounts, partner_i
             ])
         # filter on account.move.line having the same company as the statement line
         domain = expression.AND([domain, [('company_id', '=', st_line.company_id.id)]])
+        domain = expression.AND([domain, [('move_id.state', '!=', 'draft')]])
```

Comment:
you sure?? draft moves have always been reconciliable, since eons!

I think we should continue that way

Reply:
How can you reconcile with something that is bound to be changed later?

Reply:
Maybe we should re-check with avw, but I'm pretty sure he said that we shouldn't see draft moves here.